SmartDeviceLink/SDLTextAndGraphicManager.m

```diff
@@ -52,7 +52,7 @@ A show describing the current text and images on the screen (not soft buttons, e
 @property (strong, nonatomic, nullable) SDLDisplayCapabilities *displayCapabilities;
 @property (strong, nonatomic, nullable) SDLHMILevel currentLevel;
 
-@property (strong, nonatomic) SDLArtwork *blankArtwork;
+@property (strong, nonatomic, nullable) SDLArtwork *blankArtwork;
```

Comment:
This shouldn't be reset to nil

Reply:
As mentioned in issue #953, I am getting a `sdl_fileManager_fileDoesNotExistError` for the blank artwork when I try to use it on reconnects. Setting the blank artwork to `nil` on disconnect was the only way I could fix problem.

Reply:
In that case, put a comment explaining that where it's reset.